plugin/build.gradle

```diff
@@ -23,13 +20,13 @@ sourceSets {
 }
 
 dependencies {
-    compile gradleApi()
+    compile libraries.gradleApi
 
-    testCompile gradleTestKit()
-    testCompile 'junit:junit:4.12'
-    testCompile 'com.google.truth:truth:0.30'
-    testCompile 'com.google.guava:guava:19.0'
-    testCompile 'com.google.code.findbugs:jsr305:3.0.0'
+    testCompile libraries.gradleTestKit
+    testCompile libraries.junit
+    testCompile libraries.truth
+    testCompile libraries.guava
+    testCompile libraries.findbugs.annotations
```

Comment:
I took the opportunity to stretch a little this trivial example introducing things we have already seen in another real life projects - just to get a feeling of how this will scale up.
So don't mind too much the ~nonsensical name here 😅